data/human/pirate jobs.txt

```diff
@@ -3396,3 +3396,21 @@ mission "Republic Assassination [4]"
 		"reputation: Republic" -= 25
 		"reputation: Navy (Oathkeeper)" -= 25
 		"reputation: Pirate" += 10
+
+mission "Slave revolt"
+	landing
+	repeat
+	to offer
+		random < 10
+	on offer
+		require "Slave" 2
+		conversation
+			`	You are flicking through your diary in your cabin, when you hear your door start to open.`
```

Comment:
```suggestion
			`You are flicking through your diary in your cabin, when you hear your door start to open.`
```
First text line shouldn't have indent because the game already did that for some reason, but that's how it is.